projects/inspec.md

```diff
@@ -10,22 +10,32 @@ The InSpec project includes InSpec as well as several resource packs and plugins
 
 #### Project Owners
 
+James Stocks
+  - Github: [james-stocks](https://github.com/james-stocks)
+  - Slack: @Stocksy
+
+Nick Schwaderer
+ - Github: [Schwad](https://github.com/Schwad)
+ - Slack: @schwad
+
+#### Project Approvers
+
 Clinton Wolfe
   - Github: [clintoncwolfe](https://github.com/clintoncwolfe)
   - Slack: @cwolfe
 
+#### Project Reviewers
+
+#### Project Alums
+
 Ryan Davis
   - Github: [zenspider](https://github.com/zenspider)
   - Slack: @rdavis
 
-#### Project Approvers
-
-#### Project Reviewers
-
 ### Project Repositories
 
 - [inspec](https://github.com/inspec/inspec)
-- [inspec-gcp](https://github.com/inspec/inspec-gcp)
```

Comment:
If we're not maintaining this anymore we should note that on the repo as well

Reply:
The repo is listed as maintained by https://github.com/chef/chef-oss-practices/blob/master/projects/inspec-cloud.md 

I don't see the repo referencing this team/project

Reply:
k